src/DeckGlMap/layers/WmsLayer.js

```diff
@@ -0,0 +1,95 @@
+import {CompositeLayer} from '@deck.gl/core';
+import {TileLayer} from '@deck.gl/geo-layers';
+import {BitmapLayer} from '@deck.gl/layers';
+import {load} from '@loaders.gl/core';
+import SphericalMercator from '@mapbox/sphericalmercator';
+
+const conversion = new SphericalMercator({size: 512, antimeridian: true});
```

Comment:
size=512 means that tiles have size 512X512 px? Standard is 256px and 512px are used usualy on hiDPI devices.

Reply:
https://github.com/gisat-panther/ptr-maps/pull/116/commits/4c411e12e9dbd3f56f98127289559f569561c3ab